pkg/hostagent/services.go

```diff
@@ -63,6 +66,28 @@ type opflexService struct {
 	Attributes map[string]string `json:"attributes,omitempty"`
 }
 
+// type of Openshift Infrastructure Ip's
+const (
+	IngressIp string = "IngressIp"
+	DnsIp     string = "DnsIp"
+)
+
+// Name of the Openshift Service
+const (
+	RouterInternalDefult string = "router-internal-default"
+)
+
+// Namespace of Openshift Service
+const (
+	OpenShiftIngress string = "openshift-ingress"
+)
```

Comment:
This initialization should be on a list of struct (which will support the ability to add more).